packages/playwright/src/reporters/base.ts

```diff
@@ -496,7 +507,10 @@ export function formatError(screen: Screen, error: TestError): ErrorDetails {
   // Now that we filter out internals from our stack traces, we can safely render
   // the helper / original exception locations.
   const parsedStack = stack ? prepareErrorStack(stack) : undefined;
-  tokens.push(parsedStack?.message || message);
+  let parsedMessage = parsedStack?.message || message;
+  if (!screen.colors.enabled)
+    parsedMessage = stripAnsiEscapes(parsedMessage);
```

Comment:
do this once and for all in the end? I can see similar logic for snippets.

Reply:
done